content/tutorials/react-setup.md

```diff
@@ -40,7 +40,7 @@ This tutorial shows how you can build a basic [React](https://reactjs.org/) app
 5. Below the `import './App.css'` line, let's import the packages we installed, set up web3 and unlock MetaMask accounts (if locked):
 
    ```javascript
-   import { Ocean } from '@oceanprotocol/squid/dist/node/squid'
+   import { Ocean } from '@oceanprotocol/squid'
```

Comment:
@jernejpregelj This line also occurs in the code listing at the bottom of the page and on the bottom of the /tutorials/react-publish-data-set/ page.

Reply:
@ttmc pushed commit with fix.